### DISCUSSION TITLE
New dockerhub repository owncast/owncast does not have arm binaries.

### DISCUSSION THREAD

**Discussion by TinFoilFox** [owncast/owncast#2828] [Q&A] [answered]
~~In the news I saw that owncast will be changing its docker repository to `owncast/owncast` but since I'm running on Oracle's cloud servers I need an `arm64` image to run. I can compile my own images no problem in the mean time but I wanted to make sure that the new repository will still have arm images like `gabekangas/owncast` has.~~

Edit: I just saw this question has already been asked. I apologize.

Second slightly unrelated question, will there be a `develop` or `nightly` images that we could use to test out new features or do we have to build the image ourselves everytime a new feature needs testing?

**Comment by gabek** [comment, marked as answer]
Yup! https://github.com/owncast/owncast/pkgs/container/owncast/77710544?tag=nightly exists as a nightly image. It's the `develop` branch. It's the same thing that gets deployed to https://nightly.owncast.tv/ every day. I wouldn't suggest running it in production but it's great for testing.
